lib/job-simulation.rb

```diff
@@ -5,11 +5,34 @@ class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @jobs_available = jobs_available
+    @job_seekers = job_seekers
+    @workers = Stack.new
+    @waiting = Queue.new
+    n = 0
+    @jobs_available.times do
+      @workers.push("Worker ##{n+1}")
+      n += 1
+    end
```

Comment:
```
jobs_available.times do |i|
 @workers.push("Worker ##{i+1}")
end
```
This definitely works to push in `@workers` all workers like this:
"Worker #1", "Worker #2", "Worker #3", "Worker #4"......"
but if I do the same thing for `@waiting`:
```
job_seekers.times do |i|
 @waiting.enqueue("Worker ##{i+1}")
end
```
...`@waiting `will have people starting from **1 to job_seekers**:
"Worker #1", "Worker #2", "Worker #3", "Worker #4".........
because iteration variable `i` starts with **0** each time.

To make my `@waiting` list starts with  "Worker #5 " and so on, I didn't use iteration variable `|i|`,. Instead, I created a variable `n` outside the block, and starting to increment `n` inside block(with `@workers`). So when I put `n` in second block (with `@waiting`) it continue to increment from where it stopped in first block. This allows me to have `@waiting `and @`workers` like this:
`@workers = ["Worker #1", "Worker #2", "Worker #3", "Worker #4"]`
`@waiting = ["Worker #5", "Worker #6", "Worker #7", "Worker #8" ........]`